code/modules/interface/multiContext/context_actions.dm

```diff
@@ -8,6 +8,7 @@
 	var/use_tooltip = 1
 	var/close_clicked = 1
 	var/flick_on_click = null
+	var/dir = NORTH // rarely used
```

Comment:
more like unused? lmao??

Reply:
i was attempting to make the buttons show the exact pipe that would be built if the user stayed facing that direction

Reply:
so is this PR done or not then

Reply:
i removed this part